src/database/schema/ChatSchema.ts

```diff
@@ -312,7 +325,7 @@ class ChatSchema implements BaseDbSchema {
       rawJson,
       status,
       isContinuous: false,
-      type: 'regular',
+      type: data?.message?.message_type ?? data?.message?.type,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

This change seems to be adding a fallback for the `type` field to use `data?.message?.message_type` if `data?.message?.type` is undefined. This looks fine as long as the `message_type` and `type` fields are intended to serve the same purpose.


<!-- This is an auto-generated comment by OSS CodeRabbit -->